src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -85,6 +86,29 @@ public static class DataTestUtility
         public static readonly string KerberosDomainUser = null;
         internal static readonly string KerberosDomainPassword = null;
 
+        // SQL server Version
+        private static string s_sQLServerVersion;
+        private static bool s_isTDS8Supported;
+
+        public static string SQLServerVersion
+        {
+            get
+            {
+                s_sQLServerVersion ??= GetSqlServerVersion(TCPConnectionString);
+                return s_sQLServerVersion;
+            }
+        }
+
+        // Is TDS8 supported
+        public static bool IsTDS8Supported
+        {
+            get
+            {
+                s_isTDS8Supported = GetSQLServerStatusOnTDS8(TCPConnectionString);
```

Comment:
I see you've already validated the `TCPConnectionString` in the tests, but as a stand alone utility it's good to validate these properties as well and throw exception.